From 317cd1c99aa436e578cbe625aa9ef250f9187ba2 Mon Sep 17 00:00:00 2001
From: Olof Johansson <olof@lixom.net>
Date: Thu, 23 Aug 2012 09:51:29 -0700
Subject: [PATCH 2/2] ARM: omap: add dtb targets

Makes it easier to just do 'make dtbs' for whatever the kernel was
configured for, just like some other platforms.

Signed-off-by: Olof Johansson <olof@lixom.net>
---
 arch/arm/mach-omap2/Makefile.boot |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/mach-omap2/Makefile.boot b/arch/arm/mach-omap2/Makefile.boot
index b03e562..714d5bb 100644
--- a/arch/arm/mach-omap2/Makefile.boot
+++ b/arch/arm/mach-omap2/Makefile.boot
@@ -1,3 +1,8 @@
   zreladdr-y		+= 0x80008000
 params_phys-y		:= 0x80000100
 initrd_phys-y		:= 0x80800000
+
+dtb-$(CONFIG_SOC_OMAP2420)	+= omap2420-h4.dtb
+dtb-$(CONFIG_ARCH_OMAP3)	+= omap3-beagle.dtb omap3-evm.dtb
+dtb-$(CONFIG_ARCH_OMAP4)	+= omap4-panda.dtb
+dtb-$(CONFIG_ARCH_OMAP4)	+= omap4-sdp.dtb
-- 
1.7.10.4

